### DISCUSSION TITLE
2024 First Annual Soft RISC-V Systems Workshop

### DISCUSSION THREAD

**Discussion by stnolting** [stnolting/neorv32#1104] [Show and tell]
About two weeks ago, the first annual [RISC-V workshop for soft-core processor systems](https://community.riscv.org/events/details/risc-v-international-risc-v-synergy-forums-technical-talks-and-webinars-presents-2024-first-annual-soft-risc-v-systems-workshop/) took place, where I had the great honor of presenting the NEORV32. Unfortunately, there is neither a video recording nor can the presentations and posters be downloaded anywhere. So if anyone is interested, here are the slides (at least mine).

* [neorv32.nolting.soft_riscv_systems_workshop.pptx](https://github.com/user-attachments/files/17925191/neorv32.nolting.soft_riscv_systems_workshop.pptx)
* [neorv32.nolting.soft_riscv_systems_workshop.pdf](https://github.com/user-attachments/files/17925193/neorv32.nolting.soft_riscv_systems_workshop.pdf)

**Comment by mmcheraghi** [comment]
Thanks.
